lib/presentation/pages/layouts/profile/profile_title.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class ProfileTitle extends StatelessWidget {
+  const ProfileTitle({
+    super.key,
+    required this.name,
+    required this.isAttendee,
+  });
+
+  final String name;
+  final bool isAttendee;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        Text(
+          name,
+          style: Theme.of(context).textTheme.headlineSmall,
+        ),
+        if (!isAttendee)
+          const SizedBox(
+            width: 8,
+          ),
+        if (!isAttendee)
+          const Icon(
+            Icons.verified,
+            size: 18,
+            color: Color(0xFF4285f4),
```

Comment:
Create a app_colors.dart constant file and move all hard-coded colors to that file with a color name

You can use 
https://colors.artyclick.com/color-name-finder/
to find a color name for a given hex code